src/bundle/Resources/views/themes/admin/account/bookmarks/list.html.twig

```diff
@@ -82,6 +82,7 @@
                 }) }}
                 {% include '@ibexadesign/ui/component/table/table.html.twig' with {
                     headline: 'bookmark.table.header'|trans|desc('Bookmarks') ~ ' (' ~ pager.count ~ ')',
+                    class: 'ibexa-table--has-bulk-checkbox',
```

Comment:
The problem in this PR is more widespread and I guess affects every table. It is the result of the proposed change: https://github.com/ezsystems/ezplatform-admin-ui/pull/1923#discussion_r717423381
I'm not sure whether we really want to add this to every table in the system.
Moreover, even if we want to set it on every table, this should be done as a separate table component property e.g. `has_bulk_checkbox: true` to ensure table component encapsulation.